src/index.html

```diff
@@ -7,6 +7,491 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#home"
+        class="header__logo-link"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo image"
```

Comment:
It's a bad practice to use words like `image` in alt attribute, it's obvious that it's an image. Alt attribute should shortly describe what is pictured on this image